### ISSUE TITLE
Fork as official Funkwhale project

### ISSUE THREAD

**Issue by georgkrause** [apognu/otter#123]
# TL;DR: We forked the project due to inactivity: https://dev.funkwhale.audio/funkwhale/funkwhale-android

Hello Otter Community, hello @apognu,

Otter is an huge addition to the Funkwhale Ecosystem and is likely one of the most important ways to interact with a Funkwhale services for a lot of people. Its great to have and something we need to keep.

The upcoming release of Funkwhale version 1.2 will drop support for JWT Authentication, which will break Otter for every updated instance. We should work towards fixing this, but this requires some active maintenance.

I see the following options:

* @apognu, are you still available to review merge requests and merge them? In this case I would like to search for someone to implement the new authentication system to keep Otter usable.
* Maybe we can search for some additional maintainer but keep the project in your namespace. This way we can work towards compatibility with Funkwhale 1.2, too, but the project is still associated with you.
* Its possible to evaluate a movement of the project into the namespace of Funkwhale. This would make it an official project and the community can take care for it. You can still be maintainer of the project if you have time to spend on it again.

For me its important to make sure you are still interested in working on Otter whenever you have time, but on the other side I wish Otter is still usable after we released version 1.2. So hopefully we can sort this out together. :)

Alternatively I would consider forking the project into the namespace of Funkwhale. This is similar to option 3, but without support of @apognu. But this would be just the emergency solution for me. Anyway, if you are a Android or Kotlin developer or somehow interested in contributing to Otter, let me know so we can think about how to keep the project alive.

Thanks for your work and interest!

CC @agateblue

**Comment by Funkwhale** [comment]
@apognu Good evening!

Sadly there was no reaction from you for several months or weeks now. So at first and most important: I hope you are doing well or get well soon! :heart: 

The last days brought a rapid development, some people stepped up and take care for a future development of the app. There is nothing to show yet, some members of the Funkwhale Community started working on moving the project to our infrastructure. Some CI-things are already done, but nothing to announce yet.

I still hope some day you will get back to the project! If this happens, please get in touch so we can discuss how to proceed together. 

For anyone else: If you are interested in contributing to the project, get in touch with the Funkwhale Collective. As soon as we are ready for an real, public announcement I will share the locations here, too. Stay tuned! :)

**Comment by georgkrause** [comment]
@apognu Good evening!

Sadly there was no reaction from you for several months or weeks now. So at first and most important: I hope you are doing well or get well soon! heart

The last days brought a rapid development, some people stepped up and take care for a future development of the app. There is nothing to show yet, some members of the Funkwhale Community started working on moving the project to our infrastructure. Some CI-things are already done, but nothing to announce yet.

I still hope some day you will get back to the project! If this happens, please get in touch so we can discuss how to proceed together.

For anyone else: If you are interested in contributing to the project, get in touch with the Funkwhale Collective. As soon as we are ready for an real, public announcement I will share the locations here, too. Stay tuned! :)

**Comment by dvn0** [comment]
@georgkrause I'm relieved to read that this project will continue to be maintained under the auspices of the Funkwhale community. 

I would also love to see Otter in F-Droid, which was almost ready but got left hanging some months ago as well - see: https://github.com/apognu/otter/issues/10#issuecomment-792106080 

> I still hope some day you will get back to the project! If this happens, please get in touch so we can discuss how to proceed together.

I second that. Thanks for making this project @apognu!

**Comment by georgkrause** [comment]
I cannot really take care for Otter, since I don't have access to this repository and also not to the Merge Request files by apognu there. So I think we will someday try to get our Fork into F-Droid.

For further discussions of our work, please join our Channels on Matrix, our Forum or our Gitlab, I don't want to take over the issue tracker of Otter.
